packages/contentstack-seed/src/commands/cm/stacks/seed.ts

```diff
@@ -67,21 +74,25 @@ export default class SeedCommand extends Command {
       exclusive: ['org', 'name'],
       parse: printFlagDeprecation(['s', 'stack'], ['-k', 'stack-api-key']),
     }),
+    alias: flags.string({
+      char: 'a',
+      description: 'Alias of the management token',
+    }),
   };
 
   static aliases = ['cm:seed'];
 
   async run() {
     try {
       const { flags: seedFlags } = await this.parse(SeedCommand);
+      const managementTokenAlias = seedFlags.alias;
 
-      if (!isAuthenticated()) {
+      if (!isAuthenticated() && !managementTokenAlias) {
         this.error('You need to login, first. See: auth:login --help', {
```

Comment:
@akashbhosale97 Now if we are giving the option to provide a management token as well, the error message can also be changed right?

Reply:
have changed the error message